draft-kazuho-httpbis-priority.md

```diff
@@ -334,12 +334,9 @@ amend the precedence to suit it (see {{merging}}).
 The Priority header field is an end-to-end signal of the request priority from
 the client or the response priority from the server.
 
-As is the ordinary case for HTTP caching ({{?RFC7234}}), a response with a
-Priority header field might be cached and re-used for subsequent requests.
-When an origin server generates the Priority response header field based on
-properties of an HTTP request it receives, the server is expected to control the
-cacheability or the applicability of the cached response, by using header fields
-that control the caching behavior (e.g., Cache-Control, Vary).
+A server MUST NOT generate different response based on the value of the Priority
```

Comment:
It can generate a different Priority header in response, correct?

Reply:
In 9fffd45, I've clarified that we are talking about "HTTP response" but not a response header field. Does that look better?